packages/react-devtools-shared/src/devtools/views/roblox/Div.luau

```diff
@@ -0,0 +1,61 @@
+--!strict
+
+--[[
+	This file is meant to somehow facilate JS translation of div elements to Roblox.
```

Comment:
maybe call this "noDOM", rather than named after a specific lua impl?

Reply:
For now the views are very tightly coupled to Roblox so I think I'd leave it like that. It's a good idea though if we have to support another platform that would need an integrated view. Hopefully they would just have support for sockets so we could just use the regular extension and skip re-implementing a bunch of view components